Makefile

```diff
@@ -2,14 +2,24 @@ TEST?=ralph
 
 .PHONY: test flake clean coverage docs coveralls
 
-package: build-package upload-package
+release-new-version: new_version = $(shell ./get_version.sh generate)
+release-new-version:
```

Comment:
I think we could add some small comments to those 3 new commands, especially when to use build-package vs build-snapshot-package. @romcheg what do you think?